src/evaluators/ForInStatement.js

```diff
@@ -133,6 +133,7 @@ function emitResidualLoopIfSafe(
     let absStr = realm.createAbstract(
       new TypesDomain(StringValue),
       ValuesDomain.topVal,
+      0,
```

Comment:
I had that intention for a while, but I try to keep away from the serializer. Maybe in a future pull request.